File: QLCHDoDienTu-master/Model/LoaiHang.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QLCHDoDienTu {
	public class LoaiHang {
		public int IDLoaiHang { get; set; }
		public string TenLoaiHang { get; set; }

		public LoaiHang(LoaiHang lh)
		{
			this.IDLoaiHang = lh.IDLoaiHang;
			this.TenLoaiHang = lh.TenLoaiHang;
		}
		public LoaiHang(int IDLoaiHang_, string TenLoaiHang_) {
			this.IDLoaiHang = IDLoaiHang_;
			this.TenLoaiHang = TenLoaiHang_;
		}
	}
}

File: QLBHDIENTU/src/mypack/LoaiHang.java
package mypack;

public class LoaiHang {
	int idloaihang;
	String tenloaihang;
	
	public LoaiHang(LoaiHang lh){
		this.idloaihang = lh.idloaihang;
		this.tenloaihang = lh.tenloaihang;
	}
	
	public LoaiHang(int idloaihang, String tenloaihang){
		this.idloaihang = idloaihang;
		this.tenloaihang = tenloaihang;
	}
	

	public int getIDLoaiHang() {
		return idloaihang;
	}
	
	public String getTenLoaiHang() {
		return tenloaihang;
	}
	
}
